com.unity.shadergraph/Editor/Data/Graphs/GraphData.cs

```diff
@@ -2063,6 +2184,11 @@ public override void OnAfterDeserialize(string json)
                             keywordField.SetValue(node, (JsonRef<ShaderKeyword>)keyword);
                         }
 
+                        if (!string.IsNullOrEmpty(node0.m_DropdownGuidSerialized) && dropdownGuidMap.TryGetValue(node0.m_DropdownGuidSerialized, out var dropdown))
```

Comment:
shouldn't need the changes in this file -- legacy nodes should never have dropdowns to upgrade?

Reply:
I've removed this one.